INSTALL

```diff
@@ -74,15 +74,15 @@ http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTAL
 If you wish to build another version of QGIS, ensure to checkout the appropriate
 release branch. The QGIS source code can be found in the repository (https://github.com/qgis/QGIS).
 
-Please visit http://qgis.org for information on joining our mailing lists 
+Please visit http://qgis.org for information on joining our mailing lists
 and getting involved in the project further.
 
 /!\ Note to document writers: Please use this document as the central
-place for describing build procedures. Please do not remove this notice. 
+place for describing build procedures. Please do not remove this notice.
 
-/!\ Note to document writers: This document is generated from 
-doc/INSTALL.t2t - if you need to edit this document, be sure to edit that 
-file rather than the generated INSTALL document found in the root of the 
+/!\ Note to document writers: This document is generated from
+doc/INSTALL.t2t - if you need to edit this document, be sure to edit that
+file rather than the generated INSTALL document found in the root of the
 source directory.
```

Comment:
@citizenfish As mentioned here, this file is a generated one so you probably want to edit the doc/osx.t2t file

Reply:
My bad! I'll get that done